hack/scc-prep.sh

```diff
@@ -37,13 +37,14 @@ EOF
 SERVICE_ACCOUNT_NAME=default
 NAMESPACE=scc-check
 SERVER=https://127.0.0.1:6443
-SECRET_NAME=$(kubectl get serviceaccount ${SERVICE_ACCOUNT_NAME} \
-  --namespace ${NAMESPACE} \
+SECRET_NAME=$(kubectl get serviceaccount "${SERVICE_ACCOUNT_NAME}" \
+  --namespace "${NAMESPACE}" \
   -o jsonpath='{.secrets[0].name}')
 
-ca=$(kubectl get -n $NAMESPACE secret/$SECRET_NAME -o jsonpath='{.data.ca\.crt}')
-token=$(kubectl get -n $NAMESPACE secret/$SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode)
-namespace=$(kubectl get -n $NAMESPACE secret/$SECRET_NAME -o jsonpath='{.data.namespace}' | base64 --decode)
+ca=$(kubectl get -n "$NAMESPACE" "secret/$SECRET_NAME" -o jsonpath='{.data.ca\.crt}')
+token=$(kubectl get -n "$NAMESPACE" "secret/$SECRET_NAME" -o jsonpath='{.data.token}' | base64 --decode)
+# shellcheck disable=SC2034
+namespace=$(kubectl get -n "$NAMESPACE" "secret/$SECRET_NAME" -o jsonpath='{.data.namespace}' | base64 --decode)
```

Comment:
This variable was unused, is line https://github.com/openshift/microshift/pull/1606/files#diff-6a8a26c07d340b87b162acac6d93aa2110dc92ee749a8841e159a7f3b851f4e1R61 a type and was intended to be used here?

Reply:
Let's check what this script is for. We may be able to get rid of it completely.